internal/kafka/internal/environments/development.go

```diff
@@ -22,11 +22,10 @@ func NewDevelopmentEnvLoader() environments.EnvLoader {
 		"mas-sso-realm":                                   "rhoas",
 		"osd-idp-mas-sso-realm":                           "rhoas-kafka-sre",
 		"enable-kafka-external-certificate":               "false",
-		"aws-cluster-compute-machine-type":                "m5.2xlarge",
 		"allow-developer-instance":                        "true",
 		"quota-type":                                      "quota-management-list",
 		"enable-deletion-of-expired-kafka":                "true",
-		"dataplane-cluster-scaling-type":                  "manual",
+		"dataplane-cluster-scaling-type":                  "auto",
```

Comment:
I guess this got accidentally in the PR. It should be `manual`

Reply:
Yes. This slipped in from the various tests I was doing.

Good catch, will revert.